source/common/event/libevent_scheduler.cc

```diff
@@ -15,7 +15,11 @@ void recordTimeval(Stats::Histogram& histogram, const timeval& tv) {
 }
 } // namespace
 
-LibeventScheduler::LibeventScheduler() : libevent_(event_base_new()) {
+LibeventScheduler::LibeventScheduler() {
+  auto eventBase = event_base_new();
```

Comment:
event_base per style guide.

Reply:
Thanks, it should be fixed in 0c2f5f467686c2ee1afed552a24f08af1f8cad2b